releasenotes/notes/dagcircuit-remove-deprecated-bit-methods-8fb2d97ac4abdf4a.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    The deprecated methods :meth:`~qiskit.dagcircuit.DAGCircuit.qubits()` and
+    :meth:`~qiskit.dagcircuit.DAGCircuit.clbits()` have been removed. They are
```

Comment:
I would probably change this to be ``DAGCircuit.qubits()`` and ``DAGCircuit.clbits()`` mostly so there is context on the class where the methods are removed from. Or maybe something like:
```
the :class:`~qiskit.dagcircuit.DAGCIrcuit` methods ``qubits()`` and ``clbits()``
```
The links here won't actually work since there is nothing to link to post removal (it doesn't error and the formatting will look fine, the context of the methods is missing).